.travis.yml

```diff
@@ -0,0 +1,5 @@
+sudo: false
+language: node_js
+node_js:
+  - "4"
+  - "5"
```

Comment:
6 was just released like an hour ago :-)

And we still do have a setup. We still use it in a lot of modules - https://github.com/continuationlabs/cl.ci/blob/master/js.travis.yml

We can probably drop everything before 4 at this point.